src/helm/benchmark/static/benchmarking.js

```diff
@@ -1571,6 +1571,11 @@ $(function () {
     return $output;
   }
 
+  // Function to track pageviews
+  function trackPageView(pagePath) {
+    gtag("config", "G-T0MW28MP3W", { page_path: pagePath });
```

Comment:
I think you can just revert `benchmarking.js`? Looking at the default behavior: `page_location` is already set to the `location.href`, which includes the query parameters, which is what we want.

Also now "config" is evented twice - one here and one on index.html.

Reply:
That's right, the intent was the have a different config here that changes page_path, but as you mentioned it's the same anyway. Additionally, as you said we shouldn't have two configs for the same Measurement ID so I'll revert this one.